tensorflow/compiler/mlir/hlo/BUILD

```diff
@@ -1062,10 +1066,74 @@ cc_library(
         ":sink_constants_to_control_flow",
         ":test_passes",
         ":transform_unranked_hlo",
+        ":userange_analysis",
         "@llvm-project//mlir:Pass",
     ],
 )
 
+gentbl(
+    name = "mlir_buffer_transforms_pass_inc_gen",
+    compatible_with = get_compatible_with_cloud(),
+    strip_include_prefix = "include",
+    tbl_outs = [
+        (
+            "-gen-pass-decls -name BufferTransforms",
```

Comment:
Use LMHLO for the name here. The name typically gives the dialect this belongs to and in this repo I think LMHLO is the correct one.